twtr/src/test/groovy/twtr/AccountSpec.groovy

```diff
@@ -105,26 +128,31 @@ class AccountSpec extends Specification {
         sus.errors["password"] != null
     }
 
-    void "Account password variations"(String thePassword, boolean isValid, boolean passwordError) {
-        expect:
+    @Unroll
+    void "Account password variations: #description"() {
```

Comment:
No, he was saying the description of the test could be placed in Unroll or in the method name.  Doing them both is redundant.  Using #description allows us to have a nicely named test in the results.